sound/soc/sof/ipc4-topology.c

```diff
@@ -314,6 +314,7 @@ static int sof_ipc4_widget_set_module_info(struct snd_sof_widget *swidget)
 static int sof_ipc4_widget_setup_msg(struct snd_sof_widget *swidget, struct sof_ipc4_msg *msg)
 {
 	struct sof_ipc4_fw_module *fw_module;
+	uint32_t type;
```

Comment:
commit message is unclear, you are referring to LP, LL, DP. You need to describe what this means, and based on the code below I don't know what LP is (looks like a typo)?

Reply:
sure. LP = low power mode. This is a attribute of pipeline, not module.